### ISSUE TITLE
Editing a Mod Description (Only), Does not give a "Publish" or "Save Changes" button

### ISSUE THREAD

**Issue by m0lz** [Nexus-Mods/web-issues#85]
If you already have an existing mod, choose Manage, Mod Details (I looked around for "Edit Attributes" specifically, but dont seem to be able to find that option, so it seems that "Mod Details" is a replacement for "Edit Attributes") .. Instead of just editing the description it seems that we then have to go through all other sections (media, documentation, permissions, requirements etc etc) because there is only a Next button at the bottom of the page, and no Save button.

So okay if we then follow through on checking all of those aspects to the mod page, then having edited the description only, you would expect by the time you get to the last page, "Publish", there would be a Publish or Save Changes button ..

.. And there isn't. All we have is "Hide My Mod" or "Delete My Mod"

How do we save the changes we have done to the description ?

**Comment by m0lz** [comment]
Aaaagh .. It seems it did the edit without needing to save changes ..

Because I just left the new site mod page, and went back to the old site mod page to make the changes instead ..

I found it had actually changed them as I had typed them .. But also the new editor has stripped a lot of the old formatting so now it needs redoing .. And I didnt even get a chance to say make the changes or not, it just did them.

**Comment by Pickysaurus** [comment]
This is also true for the changelog editing, in an earlier version there was a save/publish button

**Comment by verenion** [comment]
Assigned PT#151365763

**Comment by iamtiz** [comment]
The 'PREV' and 'NEXT' buttons do indeed save all the changes made to the mod, before moving to the previous/next steps. We added some tooltips to make it clearer. 
There's a 'SAVE' button at the top, which saves the changes without leaving the current step. We added another 'SAVE' button at the bottom, to improve the usability.

As for the publishing, the last step can be reached by simply clicking 'Publish mod' in the step list at the top
